hearth/spec/hearth/dns/host_address_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Hearth
+  module DNS
+    describe HostAddress do
+      it 'can be initialized' do
```

Comment:
Add tests for the readers?

Reply:
In Hearth we've ignored/removed tests for attr_accessors. It's indirectly tested in HostResolver I think.